app/i18n.js

```diff
@@ -0,0 +1,20 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+const languages = {
+    en: { translation: require("./lang/en.json") }
+}
+
+i18n
+    .use(initReactI18next)
+    .init({
+        resources: languages,
+        lng: window.jitsiNodeAPI.getLocale(),
```

Comment:
is this really "lng" ?

Reply:
@saghul yes https://react.i18next.com/getting-started#basic-sample